sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/degrade/DegradeRule.java

```diff
@@ -195,12 +195,13 @@ public boolean passCheck(Context context, DefaultNode node, int acquireCount, Ob
         if (grade == RuleConstant.DEGRADE_GRADE_RT) {
             double rt = clusterNode.avgRt();
             if (rt < this.count) {
-                passCount.set(0);
+                passCount.reset();
                 return true;
             }
 
             // Sentinel will degrade the service only if count exceeds.
-            if (passCount.incrementAndGet() < rtSlowRequestAmount) {
+            passCount.increment();
+            if (passCount.sum() < rtSlowRequestAmount) {
```

Comment:
```java
passCount.increment();
if (passCount.sum() < rtSlowRequestAmount)
```

The operations are not atomic? It might lead to inaccurate calculation.

Reply:
The operations are atomic because `cells` and `base` in `LongAdder` are volatile.